services/how-to/test-net.md

```diff
@@ -0,0 +1,33 @@
+---
+description: How to get testnet ETH and other test tokens for various networks.
+sidebar_position: 2
+---
+
+# Get testnet ETH and tokens
+
+Use testnet tokens to test applications on blockchain networks without using real cryptocurrency. 
+Testnet tokens allow you to simulate transactions, deploy smart contracts, and interact with dapps in a controlled environment.
+
+:::note
+
+Use [testnet endpoints](../get-started/endpoints.md) during development and initial testing phases before moving to mainnet for production deployment. 
+
+:::
+
+
+| Testnet     | Faucet URL        | Bridge URL                    |                             
+|-------------|-------------------|-------------------------------|
+| [Arbitrum Sepolia](https://docs.arbitrum.io/launch-orbit-chain/how-tos/add-orbit-chain-to-bridge-ui)| [Arbitrum Sepolia faucet](https://faucet.quicknode.com/arbitrum/sepolia)| [Arbitrum bridge](https://bridge.arbitrum.io/)|
+| [Avalanche Fuji](https://docs.avax.network/build/tutorials/platform/fuji-workflow)| [Avalanche Fuji faucet](https://faucet.avax.network)| [Avalanche bridge](https://bridge.avax.network/)|
+| [Base Sepolia](https://docs.base.org/docs/using-base/)| [Base Sepolia faucet](https://faucet.quicknode.com/base/sepolia)| [Base bridge](https://bridge.base.org/deposit)|
+| [Blast Sepolia](https://docs.blast.io/using-blast)| [Blast Sepolia faucet](https://faucet.quicknode.com/blast/sepolia)|     |
+| [Binance Smart Chain Sepolia (BSC)](https://academy.binance.com/en/articles/connecting-metamask-to-binance-smart-chain)| [BSC Sepolia faucet](https://www.bnbchain.org/en/testnet-faucet)| [BSC bridge](https://testnet.bscscan.com/bridge)|
```

Comment:
The bridge link doesn't work. Is this an appropriate replacement? https://www.bnbchain.org/en/bnb-chain-bridges